package.json

```diff
@@ -5,11 +5,15 @@
   "type": "commonjs",
   "scripts": {
     "build": "tsc && vite build",
-    "build:local": "yarn build",
+    "build:local": "env-cmd -e local -r env/.cmdrc.json yarn build",
```

Comment:
[nit] Not a requirement, but we could switch to using env-cmd's default config filepath `.env-cmdrc(|.js|.json|)` file if we wanted to avoid repeating the path 6 times